package.json

```diff
@@ -55,7 +55,8 @@
     ]
   },
   "devDependencies": {
-    "@capacitor/cli": "1.5.2"
+    "@capacitor/cli": "1.5.2",
+    "@types/lodash": "^4.14.151"
```

Comment:
@nhatndm `lodash` should be rather on `dependencies` list. It's minor btw because we don't use `npm install -production` to optimize production build yet